src/app/src/test/kotlin/org/vaccineimpact/api/tests/controllers/AbstractControllerTests.kt

```diff
@@ -56,6 +61,31 @@ class AbstractControllerTests : ControllerTests<AbstractController>()
     }
 
     @Test
+    fun `throws error if redirect param is invalid`()
+    {
+        // Mocks
+        val parameters = mapOf(":a" to "1", ":b" to "2")
+        val context = mock<ActionContext> {
+            on { params() } doReturn parameters
+            on { username } doReturn "test.user"
+            on { queryParams("redirectUrl") } doReturn "www.redirect.com"
```

Comment:
Maybe also check we're not allowed to redirect to a non-HTTPS version of the live site?

Reply:
Ah, saw the tests further down. Never mind!